src/styles/main.scss

```diff
@@ -0,0 +1,77 @@
+$hour-width: 150px;
+$minutes-width: $hour-width * 1.5;
+$heigth: $hour-width * 0.1;
+$border-radius: 0.5 * $heigth;
+$size-center: $heigth * 1.7;
+$clock-size: $minutes-width * 2.15;
+
+@mixin circle ($size) {
+  width: $size;
+  height: $size;
+  border-radius: #{0.5 * $size};
+}
+
+@mixin square ($size) {
+  width: $size;
+  height: $size;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
move it to separate files e.g `variables, mixins, reset and keyframes`

Reply:
okay